test/asm/atomic_cmpset.c

```diff
@@ -104,71 +104,95 @@ int main(int argc, char *argv[])
 
     /* -- cmpset 32-bit tests -- */
 
-    vol32 = 42, old32 = 42, new32 = 50;
```

Comment:
This was a perfectly legitimate code. What was the complaint here ?

Reply:
This is the type of warning the compiler is throwing:

```
649 atomic_cmpset.c:107:15: warning: right-hand operand of comma expression has no effect [-Wunused-value]
650      vol32 = 42, old32 = 42, new32 = 50;
651                ^
```

I didn't dig into the POSIX C standard to see what is exactly going on here; perhaps something to do with the compiler being able to ignore the RHS of a comma operator in an assignment (since the assignment is single value)...?  🤷‍♂️